examples/rstudio/rstudio_auth

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# rstudio_auth method from https://github.com/nickjer/singularity-rstudio
+
+## Confirm username is supplied
+#if [[ $# -ne 1 ]]; then
+#  exit 1
+#fi
+#USERNAME="${1}"
+#
+## Confirm password environment variable exists (set in start_rstudio.sh)
+#if [[ -z ${RSTUDIO_PASSWORD} ]]; then
+#  exit 1
+#fi
+#
+## Read in the password from user
+#read -s -p "Password: " PASSWORD
+#echo ""
+#
+#if [[ ${USERNAME} == ${USER} && ${PASSWORD} == ${RSTUDIO_PASSWORD} ]]; then
+#  exit 0
+#else
+#  exit 1
+#fi
+exit 0
```

Comment:
Need an explanation somewhere on how this script works and why it's just exiting successfully. (And if that's all we need, we could just specify `/bin/true` instead.)